backend/pkg/config/connect_cluster.go

```diff
@@ -47,7 +47,7 @@ func (c *ConnectCluster) Validate() error {
 	}
 
 	if c.URL == "" {
-		return fmt.Errorf("url to access the Connect cluster API must be set")
+		return fmt.Errorf("url to access the KafkaConnect cluster API must be set")
```

Comment:
Minor and not necessary perhaps, should we make `ConnectCluster` explicit about "Kafka Connect" and rename it to `KafkaConnectCluster` and maybe rename the file similarly?